Client/core/CGraphStats.cpp

```diff
@@ -113,6 +111,11 @@ void CGraphStats::AddTimingPoint(const char* szName)
     if (!IsEnabled())
         return;
 
+    CGraphicsInterface* pGraphics = g_pCore->GetGraphics();
+
+    uint uiViewportWidth = pGraphics->GetViewportWidth();
+    uint uiSizeX = uiViewportWidth / 4;
```

Comment:
use `std::uint32_t` please and dont use hungarian notation.

Why `width / 4`?

Reply:
width / 4 to be used to allign properly with the graph drawing down below.